crmsh/schema.py

```diff
@@ -153,4 +153,12 @@ def rng_xpath(xpath, namespaces=None):
     return _crm_schema.rng_xpath(xpath, namespaces=namespaces)
 
 
+def compare(schema1, schema2):
+    """
+    Compare two schemas
+    Return True if schema1 less than schema2
+    """
+    schema1_ver = float(schema1.split('-')[1])
+    schema2_ver = float(schema2.split('-')[1])
```

Comment:
Doesn't it distinguish for example between "pacemaker-3.10" and "pacemaker-3.1", or tell for example "pacemaker-3.11" is higher than "pacemaker-3.2"?

Reply:
I think we should reuse this:
https://github.com/ClusterLabs/crmsh/blob/b683bf0da2c4d7167ae3cd50601b3e15b2e4a7df/crmsh/utils.py#L1560

Reply:
Yes, agree, LooseVersion is good fit here. I was thinking this last week ;)